stdlib/Dates/src/periods.jl

```diff
@@ -351,6 +354,14 @@ Base.show(io::IO,x::CompoundPeriod) = print(io, string(x))
 (-)(x::CompoundPeriod) = CompoundPeriod(-x.periods)
 (-)(y::Union{Period, CompoundPeriod}, x::CompoundPeriod) = (-x) + y
 
+(*)(x::CompoundPeriod, y::Real) = iszero(y) ? CompoundPeriod() : CompoundPeriod(x.periods * y)
+(*)(y::Real, x::CompoundPeriod) = x * y
+
+(*)(A::CompoundPeriod, B::AbstractArray) = Broadcast.broadcast_preserving_zero_d(*, A, B)
+(*)(A::AbstractArray, B::CompoundPeriod) = Broadcast.broadcast_preserving_zero_d(*, A, B)
```

Comment:
Can't the second definition just be `B * A`?

Reply:
It’s the same definition as for `Period`s:
https://github.com/JuliaLang/julia/blob/24d9eab45632bdb3120c9e664503745eb58aa2d6/stdlib/Dates/src/periods.jl#L97-L98
But the same applies there, we could change them both.